test/chi/chi-vue/expansion-panel.pug

```diff
@@ -0,0 +1,302 @@
+            | Previous
+          chi-button(color='primary' data-chi-epanel-action='next')
+            | Continue
+        div(slot='change')
+          chi-button(color='primary' variant='flat' data-chi-epanel-action='change')
+            | Change
+      h4.-pt--3 Bordered Expansion Panel
+      chi-vue-expansion-panel(step="1" title="Panel Title 2" :bordered="true" state="active" data-cy="border-expansion-panel-bordered")
+        div(slot='active')
+          .chi-epanel__subtitle
+            | Optional subtitle
+          p.chi-epanel__text
+            | Content in expansion panel (e.g. a form to select a product package)
+        div(slot='done')
+          | Use this area to present
+          br
+          | a read-only summary of what the user
+          br
+          | entered or selected in step 2.
+          br
+          | (e.g.) a package selection
+        div(slot='footer')
+          chi-button(color='primary' data-chi-epanel-action='previous')
+            | Previous
+          chi-button(color='primary' data-chi-epanel-action='next')
+            | Continue
+        div(slot='change')
+          chi-button(color='primary' variant='flat' data-chi-epanel-action='change')
+            | Change
+
+      h3.-pt--6 Slots
+      chi-vue-expansion-panel(step="1" title="Panel title 1" :bordered="true" state="done" data-cy="slot-expansion-panel-done")
+        div(slot='active')
+          .chi-epanel__subtitle
+            | Optional subtitle
+          p.chi-epanel__text
+            | Content in expansion panel (e.g. a form to select a product package)
+        div.done-slot(slot='done')
+          | Use this area to present
+          br
+          | a read-only summary of what the user
+          br
+          | entered or selected in step 2.
+          br
+          | (e.g.) a package selection
+        div(slot='footer')
+        div.-p--2.footer-start-slot(slot='footerStart')
+          | Sample footer text
+        div.footer-end-slot(slot='footerEnd')
+          chi-button(color='primary' data-chi-epanel-action='previous')
+            | Previous
+          chi-button(color='primary' data-chi-epanel-action='next')
+            | Continue
+        div.change-slot(slot='change')
+          chi-button(color='primary' variant='flat' data-chi-epanel-action='change')
+            | Change
+      chi-vue-expansion-panel(step="2" title="Panel title 2" :bordered="true" state="active" data-cy="slot-expansion-panel-active")
+        div(slot='active')
+          .chi-epanel__subtitle
+            | Optional subtitle
+          p.chi-epanel__text
+            | Content in expansion panel (e.g. a form to select a product package)
+        div.done-slot(slot='done')
+          | Use this area to present
+          br
+          | a read-only summary of what the user
+          br
+          | entered or selected in step 2.
+          br
+          | (e.g.) a package selection
+        div(slot='footer')
```

Comment:
This footer does not look good.

![image](https://user-images.githubusercontent.com/86958774/179781211-1115830b-d47f-463b-be33-d9bf4a73c89a.png)

Also, you are using this `footer-start-slot` class to find the slot. You may not need these classes, just try to find the slots using another way. Is this correct? @janibolkvadze ?

https://docs.cypress.io/guides/component-testing/slots-vue